README.md

```diff
@@ -370,4 +371,4 @@ GET /ready
 
 Both can return either a `200 OK` or a `503 Service Unavailable` status code (default: `200 OK`).
 
-It is possible to customize their behaviour by injecting an `http.AliveCheck` and/or an `http.ReadyCheck` `OptionFunc` to the HTTP component constructor.
\ No newline at end of file
+It is possible to customize their behaviour by injecting an `http.AliveCheck` and/or an `http.ReadyCheck` `OptionFunc` to the HTTP component constructor.
```

Comment:
I think that we now have a builder for the HTTP component instead of OptionFuncs, so it might make sense to mention `WithAliveCheckFunc()` and `WithReadyCheckFunc()`

Reply:
You might be right, but this diff here seems to be a whitespace change, it should not even be in this PR; I will revert this tiny change.

@tpaschalis perhaps you can make a separate PR for that improvements? It seems off-topic here for what this PR tries to do.